### ISSUE TITLE
Error updating to 1.3.1 from 1.2.0

### ISSUE THREAD

**Issue by furins** [pinax/django-notification#40]
Hello, I have this issue (on mac OS 10.10, Yosemite, Python 2.7.8 - on a virtualenv):

```
pip install django-notification --upgrade
DEPRECATION: --download-cache has been deprecated and will be removed in the future. Pip now automatically uses and configures its cache.
Collecting django-notification from https://pypi.python.org/packages/source/d/django-notification/django-notification-1.3.tar.gz#md5=a21de916c7d803c82e83d2b6b38cb37f
  Using cached django-notification-1.3.tar.gz
    /usr/local/Cellar/python/2.7.8_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'install_required'
      warnings.warn(msg)
Installing collected packages: django-notification
  Found existing installation: django-notification 1.2.0
    Uninstalling django-notification-1.2.0:
      Successfully uninstalled django-notification-1.2.0
  Running setup.py install for django-notification
    /usr/local/Cellar/python/2.7.8_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'install_required'
      warnings.warn(msg)
    error: can't copy 'notification/templates/notification': doesn't exist or not a regular file
```
[full trace on pastebin](http://pastebin.com/nBjnME3V)

I have no idea how to circumvent/fix this issue. It seems that `setup.py` is unable to find the contents of the `templates` folder. `pip` and `setuptools` are updated to the latest version. If you need any other info please do not hesitate to ask me, 
Thanks in advance!
Stefano

**Comment by casio** [comment]
Broken for me, too

**Comment by paltman** [comment]
@casio @furins fixed in 1.3.2 which just was pushed to PyPI.  Thanks for reporting this issue!
